src/course-unit/data/thunk.js

```diff
@@ -249,12 +246,7 @@ export function duplicateUnitItemQuery(itemId, xblockId) {
     dispatch(showProcessingNotification(NOTIFICATION_MESSAGES.duplicating));
 
     try {
-      const { locator } = await duplicateUnitItem(itemId, xblockId);
-      const newCourseVerticalChildren = await getCourseVerticalChildren(itemId);
-      dispatch(duplicateXBlock({
```

Comment:
[inform]: since we use legacy functionality to store xblock data, in the current implementation it makes sense to use thunks exclusively for working with the backend API, without storing data from the Redux store